xml/libvirt_configuration_virsh.xml

```diff
@@ -0,0 +1,1231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:base="libvirt_configuration_virsh.xml" version="5.0" xml:id="cha.libvirt.config_virsh">
+ <title>Configuring Virtual Machines with virsh</title>
+ <info>
+  <abstract>
+   <para>
+    In addition to the Virtual Machine Manager, the VM guest can also be configured with <command>virsh</command>.
+    <command>virsh</command> is a command line tool to manage virtual machines based on virtualization infrastructures such as
+    KVM, Xen, VMware, or LXC.
+    <command>virsh</command> can control the state of a VM, edit a the configuration of a VM or even migrate a VM to another host.
```

Comment:
Proposed rewrite:

"You can use <command>virsh</command> in addition to using the Virtual Machine Manager's ability to configure the VM guest. <command>virsh</command> is a command line tool that manages virtual machines created by VM software, such as KVM, Xen, VMWare, and LXC. <command>virsh</command> can also control the state of a VM, edit its configuration, or even migrate it to another host."

Reply:
Rewrite 2:

"You can use <command>virsh</command> to configure the VM guest in addition to using the Virtual Machine Manager. virsh is a command line tool that manages virtual machines created by VM software, such as KVM, Xen, VMWare, and LXC. virsh can also control the state of a VM, edit its configuration, or even migrate it to another host."

That's a little better.

Reply:
@bbevan : Thanks! I used your second proposal. :) But I still changed the last sentence to 'With virsh [...]' because from a linguistic POV, virsh is inanimate and cannot be the agent of a sentence.